src/app/route/MyAccountOrders/MyAccountOrders.component.js

```diff
@@ -0,0 +1,166 @@
+/**
+ * ScandiPWA - Progressive Web App for Magento
+ *
+ * Copyright © Scandiweb, Inc. All rights reserved.
+ * See LICENSE for license details.
+ *
+ * @license OSL-3.0 (Open Software License ("OSL") v. 3.0)
+ * @package scandipwa/base-theme
+ * @link https://github.com/scandipwa/base-theme
+ */
+
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import TextPlaceholder from 'Component/TextPlaceholder';
+import { Redirect } from 'react-router';
+import { Link } from 'react-router-dom';
+import './MyAccountOrders.style';
+
+// TODO remove when BE functionality to get store currency symbol is created
+
+class MyAccountOrders extends Component {
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            finishedLoading: false
+        };
+    }
+
+    componentDidMount() {
+        this.requestCustomerOrders();
+        this.updateBreadcrumbs();
+    }
+
+    requestCustomerOrders() {
+        const { requestCustomerOrders } = this.props;
+
+        return requestCustomerOrders().then(() => this.setState({ finishedLoading: true }));
+    }
+
+    updateBreadcrumbs() {
+        const { updateBreadcrumbs } = this.props;
+        const breadcrumbs = [
+            {
+                url: '/my-account-orders',
+                name: 'My Account Orders'
+            },
+            {
+                url: '/',
+                name: 'Home'
+            }
+        ];
+
+        updateBreadcrumbs(breadcrumbs);
+    }
+
+    renderOrderRows() {
+        const { orders, currency } = this.props;
+        const { finishedLoading } = this.state;
+
+        if (finishedLoading && currency) {
```

Comment:
Can you please merge this table and placeholder table into one? It is possible to use `<TextPlaceholder content={ order.id } />` and it is possible to fill an array... Also, please destruct the `order`.